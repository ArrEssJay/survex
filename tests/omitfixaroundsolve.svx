; Regression test for bug fixed in 1.2.14
*fix a
*solve
*fix b
a b 123 10.0 -01
