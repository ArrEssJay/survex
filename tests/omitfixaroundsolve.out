./omitfixaroundsolve.svx:2: warning: FIX command with no coordinates - fixing at (0,0,0)

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...
./omitfixaroundsolve.svx:2: warning: Unused fixed point "a"
./omitfixaroundsolve.svx:4: More than one FIX command with no coordinates

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 3 survey stations, joined by 1 leg.
There are 0 loops.
Survey has 2 connected components.
Total length of survey legs =  123.00m ( 123.00m adjusted)
Total plan length of survey legs =  122.98m
Total vertical length of survey legs =    2.15m
Vertical range = 2.15m (from a at 0.00m to b at -2.15m)
North-South range = 121.11m (from b at 121.11m to a at 0.00m)
East-West range = 21.36m (from b at 21.36m to a at 0.00m)
   2 1-nodes.
Done.
There were 2 warning(s) and 1 error(s) - no output files produced.
